[docs][control-plane-manager] Add info about feature gates (#16426)
deckhouse/deckhouse: modules/040-control-plane-manager/docs/README.md

- `user-authn.deckhouse.io/dex-provider` — Dex provider identifier (requires `federated:id` scope)

You can find how to set up policies in [a special FAQ section](faq.html#how-do-i-configure-additional-audit-policies).

## Feature Gates

> Feature gates can be specified by using ModuleConfig `control-plane-manager` [enabledFeatureGates](configuration.html#parameters-enabledFeatureGates) parameter.

Reviewer: ```suggestion
You can configure feature gates using the [enabledFeatureGates](configuration.html#parameters-enabledFeatureGates) parameter of the `control-plane-manager` ModuleConfig resource.
```

Changing the list of feature gates causes a restart of the component it relates to (e.g., `kube-apiserver`, `kube-scheduler`, `kube-controller-manager`, `kubelet`).

> An example of enabling feature gates `ComponentFlagz` and `ComponentStatusz`:

Reviewer: ```suggestion
The following example enables the `ComponentFlagz` and `ComponentStatusz` feature gates:
```
